### PR TITLE
Implement paging in transactions list

### LINKED ISSUE #35: Enable traversal over transaction pages
When searching for transactions on a Bitcoin address, the result is returned as a paged list. There is, however, no means to go to the next or previous page currently. This becomes extra important as the nodes can only return transactions in ascending order (oldest first) -> we can't view newer transactions.

### PR DESCRIPTION
**Pull request by laszlourszuly**
This change required significant code clean up in order to be manageable.

Fixes: #35

### CODE REVIEW

**Comment by laszlourszuly** [comment]
There is quite a lot happening here, but all of it is necessary and ultimately for the greater good. The `Add transaction paging` is the important commit here.

File: explorer/app2/js/app.js [hunk 1]
```diff
@@ -13,253 +13,382 @@ const competitors = [
...
-
-  $(tickerClass).each(function() {
-    $(this).FlipClock(diff, {
-      clockFace: 'HourlyCounter',
-      countdown: true
-    });
+function updateStats() {
+  $('.tps-spot').each(function() {
+    $(this).text(beautifulNumber(currentMetricsResult.spot));
+  });
```
Comment (MuncleUscles) [inline explorer/app2/js/app.js:162]: I might be wrong, but I think all of these can be rewritten as 
`$('.tps-spot').text(beautifulNumber(currentMetricsResult.spot));`

**Review by MuncleUscles — APPROVED** [review]
Some jquery stuff could be simplified

File: explorer/app2/js/app.js (hunk 1 above)
Comment (laszlourszuly) [reply explorer/app2/js/app.js:162]: Yes, I was hoping for that "implied iteration" to just be there, like, always. But sometimes (especially with the countdown tickers) this trick just doesn't work :( I'll look into this if/as I get a couple of hours to spare, though (we're getting dangerously close to Java levels of verbosity...)

**Review by blenessy — APPROVED** [review]

**Pull request merged**